README.md

```diff
@@ -70,6 +73,24 @@ const didDocument = await ipid.update(pem, (document) => {
 
 ## API
 
+### getDid(pem)
+
+Returns the DID associated to a private key in PEM format.
+
+#### pem
+
+Type: `String`
+
+A private key in PEM format.
```

Comment:
Should we make it clear everywhere we mention pem that we support pkcs1 and pkcs8?

Reply:
A tested with every PEM format for private keys available in [crypto-keys-composer](https://github.com/ipfs-shipyard/js-crypto-key-composer/) and it accepted all of them.

Reply:
Added `pkcs1-pem` and `pkcs8-pem` as the supported formats.